src/plugins/tabs/components/removeDialog.js

```diff
@@ -0,0 +1,34 @@
+export default class RemoveDialog {
```

Comment:
This adds a lot more complexity, than just calling openmct.overlays.dialog from tabs.vue